coalib/bearlib/abstractions/Lint.py

```diff
@@ -1,339 +0,0 @@
-import os
-import re
-import shutil
-from subprocess import check_call, CalledProcessError, DEVNULL
-import tempfile
-
-from coalib.bears.Bear import Bear
-from coalib.misc.Decorators import enforce_signature
-from coalib.misc.Shell import escape_path_argument, run_shell_command
-from coalib.results.Diff import Diff
-from coalib.results.Result import Result
-from coalib.results.RESULT_SEVERITY import RESULT_SEVERITY
-
-
-class Lint(Bear):
```

Comment:
Yes this seems sane. There are a lot of WIP bears right now, once we reach 0 PRs with old lint and all existing bears having new `Lint`, we can delete this.